CMakeLists.txt

```diff
@@ -113,15 +113,14 @@ else()
   set(HAS_OG_FLAG 0)
 endif()
 
-# Set custom build flags for RelWithDebInfo.
-# -DNDEBUG purposely omitted because we want assertions.
+#
+# Build-type: RelWithDebInfo
+#
 if(HAS_OG_FLAG)
-  set(CMAKE_C_FLAGS_RELWITHDEBINFO "-Og -g"
-    CACHE STRING "Flags used by the compiler during release-with-debug builds." FORCE)
-elseif(NOT MSVC)
-  set(CMAKE_C_FLAGS_RELWITHDEBINFO "-O2 -g"
```

Comment:
`-O2 -g` appears to be the CMake default for RelWithDebInfo, so I removed this redundant step.